app/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyController.scala

```diff
@@ -996,7 +996,7 @@ class CreateCaseJourneyController @Inject() (
             generatedAt.ddMMYYYYAtTimeFormat,
             caseSLA,
             controller.downloadCreateCaseConfirmationReceipt,
-            controller.downloadCreateCaseConfirmationReceiptAsPdf(s"case-$caseReferenceId.pdf"),
+            controller.downloadCreateCaseConfirmationReceiptAsPdf(s"Document_receipt_$caseReferenceId.pdf"),
```

Comment:
Can we make it playback the **entry number** instead of the case reference number?